css/datagrid/datagrid.css

```diff
@@ -93,7 +93,6 @@
 
 .highcharts-datagrid-container {
     display: block;
-    min-height: 200px;
     font-family: 1em;
```

Comment:
I found this earlier but just remembered now. I know it's not your line, but it's worth fixing as soon as possible, not to miss it again. :)